app/controllers/api/databases.php

```diff
@@ -2790,3 +2873,98 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
+            'total' => $dbForProject->count('slowQueries', Query::groupByType($queries)['filters'], APP_LIMIT_COUNT),
+        ]), Response::MODEL_DOCUMENT_LIST);
+    });
+
+App::get('/v1/databases/:databaseId/slow-queries/:documentId')
+    ->desc('Get Slow Query Document')
+    ->groups(['api', 'database'])
+    ->label('docs', false)
+    ->label('scope', 'documents.read')
+    ->label('usage.metric', 'documents.{scope}.requests.read')
+   // ->label('usage.params', ['databaseId:{request.databaseId}', 'collectionId:{request.collectionId}'])
+    ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN])
+    ->label('sdk.namespace', 'databases')
+    ->label('sdk.method', 'getSlowQuery')
+    ->label('sdk.description', '/docs/references/databases/get-slow-query.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_DOCUMENT)
+    ->param('databaseId', '', new UID(), 'Database ID.')
+    ->param('documentId', '', new UID(), 'Document ID.')
+    ->inject('response')
+    ->inject('dbForProject')
+    ->action(function (string $databaseId, string $documentId, Response $response, Database $dbForProject) {
+        $document = $dbForProject->getDocument('slowQueries', $documentId);
+        if ($document->isEmpty() || $document->getAttribute('databaseId') !== $databaseId) {
+            throw new Exception(Exception::DOCUMENT_NOT_FOUND);
+        }
+
+        $response->dynamic($document, Response::MODEL_DOCUMENT);
+    });
+
+App::delete('/v1/databases/:databaseId/slow-queries/:documentId')
+    ->desc('Delete Slow query Document')
+    ->desc('List  Documents')
+    ->groups(['api', 'database'])
+    ->label('docs', false)
+    ->label('scope', 'documents.write')
+    ->label('usage.metric', 'documents.{scope}.requests.delete')
+    //->label('usage.params', ['databaseId:{request.databaseId}', 'collectionId:{request.collectionId}'])
+    ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN])
+    ->label('sdk.namespace', 'databases')
+    ->label('sdk.method', 'deleteSlowQuery')
+    ->label('sdk.description', '/docs/references/databases/delete-slow-query-document.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_NOCONTENT)
+    ->label('sdk.response.model', Response::MODEL_NONE)
+    ->param('databaseId', '', new UID(), 'Database ID.')
+    ->param('documentId', '', new UID(), 'Document ID.')
+    ->inject('response')
+    ->inject('dbForProject')
+    ->action(function (string $databaseId, string $documentId, Response $response, Database $dbForProject) {
+        $document = $dbForProject->getDocument('slowQueries', $documentId);
+        if ($document->isEmpty() || $document->getAttribute('databaseId') !== $databaseId) {
+            throw new Exception(Exception::DOCUMENT_NOT_FOUND);
+        }
+        $dbForProject->deleteDocument('slowQueries', $documentId);
+        $dbForProject->deleteCachedDocument('slowQueries', $documentId);
```

Comment:
We don't need to clear the cached document here, `deleteDocument` does it internally

Reply:
done!